com.microsoft.mrtk.uxcore/Interop/UGUIInputAdapter.cs

```diff
@@ -143,6 +122,24 @@ protected override void Awake()
                 InteractionManager.interactorRegistered += OnInteractorRegistered;
                 InteractionManager.interactorUnregistered += OnInteractorUnregistered;
             }
+
+            // Some interactors (all?) get registered before we have a chance to subscribe to the
+            // registration events. As a result; let's manually scrape the manager for the proxy interactor.
+            if (proxyInteractor == null && ThisInteractable != null && InteractionManager != null)
+            {
+                // Go find the proxy interactor that this shim is associated with.
+                interactorQueryList.Clear();
+                InteractionManager.GetRegisteredInteractors(interactorQueryList);
+
+                foreach (var interactor in interactorQueryList)
+                {
+                    if (interactor is IProxyInteractor)
+                    {
+                        proxyInteractor = interactor as IProxyInteractor;
```

Comment:
Could reduce this to one cast (pending naming of the temp variable 🙃)

```suggestion
                    if (interactor is IProxyInteractor iProxyInteractor)
                    {
                        proxyInteractor = iProxyInteractor;
```

Reply:
Fixed, but with less gross name :D